src/main/java/parser/Parser.java

```diff
@@ -78,6 +81,16 @@ private static String parseAnswer(String arg) throws InvalidInputException {
         return answer;
     }
 
+    private static Command prepareRevise(String arg) throws InvalidInputException {
```

Comment:
It would be better if "commandArgs" is used instead of "arg" to standardize with the rest